third_party/packages/cupertino_icons/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ name: cupertino_icons
 description: Default icons asset for Cupertino widgets based on Apple styled icons
 repository: https://github.com/flutter/packages/tree/master/third_party/packages/cupertino_icons
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+cupertino_icons%22
-version: 1.0.3
+version: 1.0.3+1
```

Comment:
I updated to `1.0.3+1` since it doesn't impact the public API and we want it published to update https://pub.dev/packages/cupertino_icons, but I can swap to 1.0.4 if preferred.

Reply:
`publishable` answered this for me, changed to 1.0.4.